### PR TITLE
feat: Allow custom icon size

### PR DESCRIPTION
**Pull request by belgamo**
### Motivation
Add missing feature #1659

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -157,6 +161,7 @@ class Button extends React.Component<Props, State> {
       dark,
       loading,
       icon,
+      iconSize = 16,
```
Comment (erickcelio) [inline src/components/Button.tsx]: What do you think about put that default value on defaultProps ?

Comment (belgamo) [reply src/components/Button.tsx]: I think it's better! Thank you!

**Comment by taeingyu** [comment]
any updates?

**Comment by Dgrady3** [comment]
Can we please get an update on this issue?

**Comment by Dgrady3** [comment]
@belgamo thank you for updating the PR! 🙇‍♂️ 
Could someone here please approve and click merge to master?

**Comment by JoaoLSA** [comment]
Is it working?

**Comment by Trancever** [comment]
Icons in `Button` will have the same size as the text. You can customize it by passing `fontSize`.

**Comment by joaosousafranco** [comment]
> Icons in `Button` will have the same size as the text. You can customize it by passing `fontSize`.

Is this because it is a material convention or is there any technical limitation? I have a design where the icon has different size than the font size, ill probably need to do a custom button.

**Comment by yovany-lg** [comment]
In version 3.10.1 you can do something like:
`
      <Button
        icon="shopping"
        mode="outlined"
        onPress={() => console.log('Pressed')}
        style={styles.container}
        labelStyle={{ color: theme.primary, fontSize: 22 }}
        color={theme.primary}
      >
        <Text style={textStyle.text}>Add To Cart</Text>
      </Button>
`
it worked for me.

**Comment by joaosousafranco** [comment]
> In version 3.10.1 you can do something like:
> `<Button icon="shopping" mode="outlined" onPress={() => console.log('Pressed')} style={styles.container} labelStyle={{ color: theme.primary, fontSize: 22 }} color={theme.primary} > <Text style={textStyle.text}>Add To Cart</Text> </Button>`
> it worked for me.

Worked perfectly for me also, thank you very much.

**Pull request closed without merging**